prime-angular-frontend/src/app/modules/enrolment/shared/components/obo-site-form/obo-site-form.component.html

```diff
@@ -32,7 +32,7 @@
         </div>
       </div>
 
-      <app-address [form]="site.get('physicalAddress')"
+      <app-address [form]="site?.get('physicalAddress')"
```

Comment:
When the component it shouldn't be possible to have an site FormGroup that is undefined or null.